### ISSUE TITLE
train set mean

### ISSUE THREAD

**Issue by yzcjtr** [mrharicot/monodepth#71]
Hi Clément,
I'm confused about the 'Train set mean' result in your paper. It is in the first row of Table 2. Could you please explain the meaning and how to calculate these values?
Lots of thanks!

**Comment by mrharicot** [comment]
Hi,
We simply averaged the depthmaps from the training set and used this mean to calculate the error on the test set.

**Comment by yzcjtr** [comment]
Thanks for your prompt reply. Seemingly this is like a measurement of distribution gap between training and testing set?

**Comment by mrharicot** [comment]
The idea was mostly to show that some of the metrics aren't great, where doing the simplest thing could still lead to reasonably good scores.

**Comment by yzcjtr** [comment]
Okay I get it. That does speak a lot about the problem. Nice trick, and thx again.
Good luck!
